src/Imdb/Title.php

```diff
@@ -581,13 +581,14 @@ public function movie_recommendations()
                     $movie['title'] = utf8_decode(trim($get_link_and_name->item(0)->nodeValue));
                     $movie['imdbid'] = $ref[1];
                     $get_rating = $xp->query(".//span[contains(@class, 'ipc-rating-star--imdb')]", $cell);
-                    if (!empty($get_rating)) {
+                    if (!empty($get_rating) && $get_rating->item(0) != null) {
```

Comment:
To keep things a little more readable can this be 
```suggestion
                    if (!empty($get_rating->item(0))) {
```
$get_rating will only be false of the xpath is invalid, which it won't be, or the dom is invalid, which also won't be true (as far as i know)